varinfo/var_info.py

```diff
@@ -589,7 +542,11 @@ def save_variable(output, group_path, element):
         all_variables = {}
 
         self.traverse_elements(
-            self.dataset, set(DAP4_TO_NUMPY_MAP.keys()), save_variable, all_variables
+            self.dataset,
+            set(DAP4_TO_NUMPY_MAP.keys()),
+            save_variable,
+            all_variables,
+            '/',
```

Comment:
The difference here is to ensure that the parsing of the DMR document also includes the root group (and assigns it to `VarInfoFromDmr.groups['/']`).